api/android/api/src/main/java/org/nnsuite/nnstreamer/SingleShot.java

```diff
@@ -168,6 +190,23 @@ public void setTimeout(int timeout) {
         }
     }
 
+    /**
+     * Sets the information (tensor dimension, type, name and so on) of input data for the given model.
+     * Updates the output information for the model internally.
+     *
+     * @param in The input tensors information
+     *
+     * @throws IllegalStateException if failed to set the input information
+     * @throws IllegalArgumentException if given param is null
+     */
+    public void setInputInfo(@NonNull TensorsInfo in) {
```

Comment:
plz add condition : throws IllegalArgumentException if in is null (line 200)

Reply:
Updated.